### PR TITLE
Update setup scripts

### PR DESCRIPTION
**Pull request by kk-min**
Update the scripts in `scripts` folder to install necessary dependencies.

## Changes
- Add `gcr` and `cloudflare` folder for initializing gcloud and wrangler CLI tools
- Update `scripts/setup.sh` to install various dependencies necessary (serverless, azure plugin, gcloud, wrangler, node16, etc.)

@ypwong99 If Azure/Alibaba deployment requires any particular installations and setup, feel free to add to this branch or create another one if we merge this branch first.

### CODE REVIEW

File: scripts/setup.sh [hunk 1]
```diff
@@ -1,6 +1,49 @@
...
+# Install Node
+NODE_MAJOR=16
+sudo apt-get install nodejs -y
+
+# Install npm and serverless
+sudo apt install npm
+sudo npm install -g serverless serverless-azure-functions --save-dev
+
+# Install gcloud CLI
+sudo apt-get install google-cloud-cli
```
Comment (ypwong99) [inline scripts/setup.sh:42]: Since this is specific to GCR, I was wondering if there was a reason we don't put this under `grc/setup.sh` instead?

**Review by ypwong99 — APPROVED** [review]
Hi @kk-min, to clarify if I'm right, these installation scripts are only used when we want to run the STeLLAR client for experiments on our own machine or cloud VMs right? As opposed to the setup that exists in GitHub Actions that are used by the runners only?

**Comment by ypwong99** [comment]
> @ypwong99 If Azure/Alibaba deployment requires any particular installations and setup, feel free to add to this branch or create another one if we merge this branch first.

Sure, I can create another PR once I finish up on #306 soon

File: scripts/setup.sh (hunk 1 above)
Comment (kk-min) [reply scripts/setup.sh:42]: Good idea, I will shift it to `gcr/setup.sh`.

**Comment by kk-min** [comment]
> Hi @kk-min, to clarify if I'm right, these installation scripts are only used when we want to run the STeLLAR client for experiments on our own machine or cloud VMs right? As opposed to the setup that exists in GitHub Actions that are used by the runners only?

You are correct, these scripts are helping users set up the tools on their own machines. For Github Actions, the same commands are run as steps where applicable.

**Pull request merged**